### PR TITLE
Fix predefined status buttons

### LINKED ISSUE #39926: [BITV] Increase contrast on focus state of "Status Message"
Unfortunately, I've noticed another issue. The elements in the "Status Message" list receive a poorly noticeable keyboard focus. This needs to be changed.
CleanShot 2023-08-16 at 20 04 46@2x

_Originally posted by @michaelnissenbaum in https://github.com/nextcloud/server/issues/37011#issuecomment-1681062093_

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Resolves: https://github.com/nextcloud/server/issues/39926

## Summary

- Add logic for predefined status to be visible.
- Add styles for radio buttons to be visible in checked, active and focus-visible state.
- Styles are same as for online status radio buttons https://github.com/nextcloud/server/pull/39332

|  Before   | After                                                        |
|-------------|---------------------------------------------------------------|
| Peek 2023-08-21 14-24 | Peek 2023-08-22 13-16

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by jancborchardt** [comment]
@marcoambrosini what do you think of the style?

File: apps/user_status/src/components/PredefinedStatusesList.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
 			:icon="status.icon"
 			:message="status.message"
 			:clear-at="status.clearAt"
-			:selected="!isCustomStatus && lastSelected === status.id"
```
Comment (ShGKme) [inline apps/user_status/src/components/PredefinedStatusesList.vue:33]: Is was the only usage of `isCustomStatus`. I think, we can now remove this prop at all, and remove passing the value from the parent

**Review by ShGKme — APPROVED** [review]

**Review by marcoambrosini — COMMENTED** [review]
I think that the border is fine, but I think we should put the radio button symbol too @JuliaKirschenheuter 
I'm afraid that we're confusing people a lot with these radio buttons that don't look like radios.

**Comment by JuliaKirschenheuter** [comment]
> I'm afraid that we're confusing people a lot with these radio buttons that don't look like radios.
yes, i'm agree that this buttons are not really radio buttons. But let them be radio buttons for now. We had only an issue  that they are not really recognizable.

> I think that the border is fine, but I think we should put the radio button symbol too
I think then it will be lots of "flashing elements". A11y-wise it is not required.

If border is ok - could we leave it like this for now? All improvements we can do afterwards

**Review by marcoambrosini — APPROVED** [review]
@JuliaKirschenheuter let's add a normal border radius and get this in, going forward I think we should add the radio button (which is now `.hidden-visually`) to all radio buttons

Screenshot 2023-08-22 at 19 47 52

**Comment by JuliaKirschenheuter** [comment]
> @JuliaKirschenheuter let's add a normal border radius and get this in

do you mean the same border radius like in "online status"?

**Comment by marcoambrosini** [comment]
> do you mean the same border radius like in "online status"?

Yes @JuliaKirschenheuter `var(--border-radius-large)`

**Pull request merged**